### PR TITLE
Added Modify Employee functionality for Admin

### PR DESCRIPTION
**Pull request by Arshpreet-Singh38**
Please review the function for the admin to modify employee details. Based on the user input the different employee details can be modified.

### CODE REVIEW

File: src/employee_management.c [hunk 1]
```diff
@@ -489,3 +490,362 @@ int emp_management(int i,int emp_id)
...
+    printf("                Press 10 Month of joining\n");
+    printf("                Press 11 Year of joining\n");
+	printf("                Press 12 Status\n");
+	printf("                Press 13 Exit\n");
+	scanf("%d",&i);
+	switch(i){
+	    case 1:{
+			
+			printf("Please provide the first name: \n");
+	    	scanf("%s",emp.f_name);
```
Comment (jagdeepbaidwan) [inline src/employee_management.c]: You can add not empty validation here

File: src/employee_management.c [hunk 2]
```diff
@@ -489,3 +490,362 @@ int emp_management(int i,int emp_id)
...
+			else{
+				printf("not connected");
+				printf("%s\n", mysql_error(conn2));
+			}	
+			break;
+		}
+		
+		case 2:{
+			printf("Please provide the last name: \n");
+	    	scanf("%s",emp.l_name);
```
Comment (jagdeepbaidwan) [inline src/employee_management.c]: You can also add notempty validation for last name too

File: src/employee_management.c [hunk 3]
```diff
@@ -489,3 +490,362 @@ int emp_management(int i,int emp_id)
...
+				printf("not connected");
+				printf("%s\n", mysql_error(conn2));
+			}
+				
+			break;
+		}
+				
+		case 7:{
+			printf("Enter gender Male,Female or Others\n");
+	    	scanf("%s",emp.gender);
```
Comment (jagdeepbaidwan) [inline src/employee_management.c]: Here you can add validation that gender should only be Male, Female or others

**Review by jagdeepbaidwan — COMMENTED** [review]
Please have a look at these suggestions

**Review by varinderdhanda — APPROVED** [review]
Its perfect now.

**Review by puneetsingh3-sysc — APPROVED** [review]
Looks good now.

**Pull request merged**